libs/community/langchain_community/vectorstores/azuresearch.py

```diff
@@ -72,8 +72,9 @@
 
 def _get_search_client(
     endpoint: str,
-    key: str,
+    key: Optional[str],
```

Comment:
Does mypy let you have an Optional before a non-Optional? I would think that it'd need a default in that case or something.

Reply:
It does, but I could add an empty string as default here? I think it doesn't let you pass something with a default in before non-default.